Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -534,18 +534,20 @@ DurationUnitRegex: !nestedRegex
 SuffixAndRegex: !simpleRegex
   def: (?<suffix>\s*(en|ën)(\s*een)?\s*(?<suffix_num>hal(f|ve)|kwart|kwartier)|(?<suffix_num>(een\s+)?kwartier))
 PeriodicRegex: !simpleRegex
-  def: \b(?<periodic>dagelijks|maandelijks|wekelijks|twee-wekelijks|jaarlijks)\b
+  def: \b(?<periodic>dagelijks|(drie)?maandelijkse?|wekelijks|twee-?wekelijks|jaarlijks|kwartaal)\b
 EachUnitRegex: !nestedRegex
-  def: (?<each>(iedere|elke)(?<other>\s+andere)?\s*{DurationUnitRegex})
-  references: [ DurationUnitRegex ]
+  def: (?<each>((iedere|elke|eenmaal per)(?<other>\s+andere)?\s*{DurationUnitRegex})|(({DurationUnitRegex}|{WeekDayRegex})\s+om(\s+de)?(?<other>\s+andere)?\s*(week|{DurationUnitRegex})))
+  references: [ DurationUnitRegex, WeekDayRegex ]
 EachPrefixRegex: !simpleRegex
-  def: \b(?<each>(iedere|elke)\s*$)
+  def: \b(?<each>(iedere|elke|eenmaal per)\s*$)
 SetEachRegex: !simpleRegex
-  def: \b(?<each>(iedere|elke)\s*)
+  def: \b(?<each>(iedere|elke|om de)\s*(?<other>\s+andere)?\s*(week)?)
 SetLastRegex: !simpleRegex
   def: (?<last>volgende?|komende|aankomende|aanstaande|deze|huidige|aanstaande|vorige?|verleden|laatste)
 EachDayRegex: !simpleRegex
-  def: ^\s*(elke)\s*dag\b
+  def: ^\s*(iedere|elke)\s*dag\b
+BeforeEachDayRegex: !simpleRegex
```

Comment:
Can you add a comment in the YAML here to explain this one?

Reply:
added.